### PR TITLE
fill upperIntermediate

### PR DESCRIPTION
**Pull request by vlad12345123**

### CODE REVIEW

File: exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs [hunk 1]
```diff
@@ -2507,6 +2508,909 @@ public static ModelBuilder FillIntermediateWithData(this ModelBuilder builder)
...
+            };
+            #endregion
+
+            #region UppedIntermediateTopics
+            var topic1 = new Question
+            {
+                Id = 401,
+                QuestionType = QuestionType.Topic,
+                Text = "What do you think about happiness",
+                LevelType = LevelType.Elementary
```
Comment (ElenaBurak) [inline exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs]: LevelType does not match

File: exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs [hunk 2]
```diff
@@ -2507,6 +2508,909 @@ public static ModelBuilder FillIntermediateWithData(this ModelBuilder builder)
...
+            };
+
+            var listeningQuestion10 = new Question
+            {
+                Id = 95,
+                FileUrl = null,
+                QuestionType = QuestionType.Audition,
+                Text = "What are the people discussing?",
+                LevelType = LevelType.Elementary
```
Comment (ElenaBurak) [inline exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs]: LevelType does not match

File: exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs [hunk 3]
```diff
@@ -2507,6 +2508,909 @@ public static ModelBuilder FillIntermediateWithData(this ModelBuilder builder)
...
+            };
+
+            var grammarQuestion10 = new Question
+            {
+                Id = 85,
+                FileUrl = null,
+                QuestionType = QuestionType.Grammar,
+                Text = "Steven _____ the wallet.",
+                LevelType = LevelType.Elementary
```
Comment (ElenaBurak) [inline exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs]: LevelType does not match

File: exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs [hunk 4]
```diff
@@ -2507,6 +2508,909 @@ public static ModelBuilder FillIntermediateWithData(this ModelBuilder builder)
...
+        public static ModelBuilder FillUpperIntermediateLevelWithData(this ModelBuilder builder)
+        {
+            #region UpperIntermediateQuestions
+            var grammarQuestion1 = new Question
+            {
+                Id = 76,
+                FileUrl = null,
+                QuestionType = QuestionType.Grammar,
+                Text = "The taxi will be here in a couple of minutes. We _____ get ready to go.",
+                LevelType = LevelType.Elementary
```
Comment (ElenaBurak) [inline exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs]: Level

File: exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs (hunk 3 above)
Comment (romanstat) [reply exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs]: upperintermediate why resolve if not

**Review by romanstat — APPROVED** [review]

**Review by artemgael96 — APPROVED** [review]

**Pull request merged**